src/pages/restaurant/reducers/restaurantReducer.js

```diff
@@ -0,0 +1,17 @@
+import { FETCH_RESTAURANT } from "../actions/types";
+
+const INITIAL_STATE = {};
+
+export const restaurantReducer = (state = INITIAL_STATE, action) => {
+  let restaurantDetails, dishItems;
+  if (action?.payload) {
+    [restaurantDetails, dishItems] = action.payload;
+  }
+
+  switch (action.type) {
+    case FETCH_RESTAURANT:
```

Comment:
you can name it set_restaurant data or something because it is not fetching something it is setting and in redux dev tools we could easily understand that we have set the store's state